index.html

```diff
@@ -529,7 +529,7 @@ <h3>BitstringStatusListCredential</h3>
               </td>
             </tr>
             <tr>
-              <td>credentialSubject.ttl</td>
+              <td id="ttl">credentialSubject.ttl</td>
```

Comment:
I flagged this elsewhere -- but don't think `ttl` works as specified and we don't need it here. I won't object to it being added here, but we'll then just need to remove it later unless it can be shown that it works properly.

Reply:
Please raise an issue if you haven't done so already noting why TTL doesn't work.

Reply:
Nevermind, I found it: https://github.com/w3c/vc-bitstring-status-list/issues/73#issuecomment-1607874866